packages/legacy/core/App/screens/PINEnter.tsx

```diff
@@ -490,34 +526,31 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
           ) : (
             displayHelpText()
           )}
-          <Text style={style.subText}>{`${
-            usage === PINEntryUsage.PINCheck ? t('PINEnter.AppSettingChangedEnterPIN') : t('PINEnter.EnterPIN')
-          }`}</Text>
+          <Text style={style.subText}>
+            {inputLabelText[usage]}
+            {usage === PINEntryUsage.ChangeBiometrics && <Text style={style.parenthesisText}>{` `}{t('PINEnter.ChangeBiometricsInputLabelParenthesis')}</Text>}
+          </Text>
           <PINInput
             onPINChanged={(p: string) => {
               setPIN(p)
               if (p.length === minPINLength) {
                 Keyboard.dismiss()
               }
             }}
-            testID={testIdWithKey(usage === PINEntryUsage.PINCheck ? 'AppSettingChangedEnterPIN' : 'EnterPIN')}
-            accessibilityLabel={
-              usage === PINEntryUsage.PINCheck ? t('PINEnter.AppSettingChangedEnterPIN') : t('PINEnter.EnterPIN')
-            }
+            testID={testIdWithKey(inputLabelText[usage])}
```

Comment:
A label is being used here rather than a testID, I think this is why the snapshot had that change I commented on.